tests/attacks/test_malware_attack.py

```diff
@@ -0,0 +1,448 @@
+# MIT License
```

Comment:
Could you please change the name of this file to `test_pe_malware_attack.py` or remove `pe` from the attack's module file name?